docs/release-notes/About.md

```diff
@@ -0,0 +1,62 @@
+﻿---
+category: Release Notes
+categoryindex: 600
+index: 1
+title: About
+---
+
+# About
+
+The release notes for the [F\# language](./Language.md), [FSharp.Core](./FSharp.Core.md) and [FSharp.Compiler.Service](./FSharp.Compiler.Service.md) are based on the [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) format.  
+The target audience of these release notes are the respective end-users.
+
+## Writing a changelog entry
+
+In order to keep the change logs consistent the following format was proposed for each entry:
+
+```md
+* <Informative description>. ([PR #16106](https://github.com/dotnet/fsharp/pull/16106))
+```
+
+Some tips:
+
+* Use valid [Markdown](https://www.markdownguide.org/).
+* Use `*` as bullet point symbol. We don't want to mix `*` and `-`.
+* Start your description with a capital and end the sentence with a dot.
+* **Always** include a link to your pull request before the closing `)`, `([PR #16106](https://github.com/dotnet/fsharp/pull/16106))`.
+* If possible, include a link to an issue on `dotnet/fsharp` use `Issue #number` before the link to the pull request.  
+
+Example:
+
+```md
+* Correctly handle assembly imports with public key token of 0 length. ([Issue #16359](https://github.com/dotnet/fsharp/issues/16359), [PR #16363](https://github.com/dotnet/fsharp/pull/16363))
+```
+
+* If possible, include a link to a language suggestion from  `dotnet/fsharp` use `Language suggestion #number` before the link to the pull request.
+
+Example:
+
+```md
+* `while!` ([Language suggestion #1038](https://github.com/fsharp/fslang-suggestions/issues/1038), [PR #14238](https://github.com/dotnet/fsharp/pull/14238))
+```
+
+* Choose the right section for your type of change. (`## Added`, `## Changed`, `## Deprecated`, `## Removed`, `## Fixed` or `## Security`).
```

Comment:
We probably don't want a separate "Security" area, they fit under "Fixes". Also since we categorizing changes, it would be great to somehow mark breaking changes (or rather ask people to mark them (**`Breaking`** prefix?)

Reply:
Hmm, I'm on the fence with the breaking prefix. Could you give me an example of something that changed, wasn't a fix and isn't breaking? I don't think that combo happens often. If it is listed under change, you can typically assume the change is breaking.

Reply:
> Breaking is a good idea, although AFAIU we don't really have a definition for breaking changes yet.

We do:
  - Any binary change in API is, well, a breaking change.
  - Any new `Obsolete`d API is a breaking change.
  - Any new warning on by default is a breaking change.
  - Almost any new change in whitespace parsing is a breaking change.
  - Any change in runtime behaviour is a breaking change (if it's not an implementation detail).
  
In other words, for libraries - if it doesn't compile anymore with no code changes, it's a breaking change. For compiler - if it's producing any new warnings - it's a breaking change. For FSharp.Core - if it produces different values in runtime than before - it's a breaking change.

Reply:
Nice summary. Worth putting somewhere maybe.